postgres/metadata.csv

```diff
@@ -6,10 +6,10 @@ postgresql.disk_read,gauge,,block,second,The number of disk blocks read in this
 postgresql.buffer_hit,gauge,,hit,second,"The number of times disk blocks were found in the buffer cache, preventing the need to read from the database. ",1,postgres,buff hit,
 postgresql.rows_returned,gauge,,row,second,The number of rows returned by queries in this database,0,postgres,rows ret,
 postgresql.rows_fetched,gauge,,row,second,The number of rows fetched by queries in this database,0,postgres,rows fetch,
-postgresql.rows_inserted,gauge,,row,second,The number of rows inserted by queries in this database,0,postgres,rows insrt,
-postgresql.rows_updated,gauge,,row,second,The number of rows updated by queries in this database,0,postgres,rows updt,
-postgresql.rows_deleted,gauge,,row,second,The number of rows deleted by queries in this database,0,postgres,rows del,
-postgresql.database_size,gauge,,byte,,The disk space used by this database.,0,postgres,db size,
+postgresql.rows_inserted,gauge,,row,second,Enabled with `relations`. The number of rows inserted by queries in this database,0,postgres,rows insrt,
+postgresql.rows_updated,gauge,,row,second,Enabled with `relations`. The number of rows updated by queries in this database,0,postgres,rows updt,
+postgresql.rows_deleted,gauge,,row,second,Enabled with `relations`. The number of rows deleted by queries in this database,0,postgres,rows del,
+postgresql.database_size,gauge,,byte,,Enabled with `collect_database_size_metrics`. The disk space used by this database.,0,postgres,db size,
```

Comment:
See Emma's comment here, this is enabled by default so I don't think this is necessary: https://github.com/DataDog/integrations-core/pull/16422/files#r1425983746